js/charts/heatmap.js

```diff
@@ -51,7 +51,7 @@ function heatmap() {
         top: 30,
         right: 15,
         bottom: 15,
```

Comment:
Pls do to not mix css and js code...better to place in css file

Reply:
its not css its js